common.go

```diff
@@ -22,6 +21,9 @@ const (
 	RecordTypeApplicationData RecordType = 23
 )
 
+const tls10Version uint16 = 0x0301
+const tls12Version uint16 = 0x0303
```

Comment:
Suggest changing the `var` block above to `const` and just adding these up there.